src/core/updateWebpackConfig.js

```diff
@@ -13,31 +13,25 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     {
       test: /\.jsx?$/,
       include: srcFolder,
-      loader: 'babel',
+      loader: 'babel-loader',
       query: {
+        babelrc: false,
         presets: [
           'es2015',
           'react',
         ],
```

Comment:
Transpiling is actually the only possible option. Relying on loaders *will* cause issues.